### DISCUSSION TITLE
The "Road To GA" Project Board

### DISCUSSION THREAD

**Discussion by shaneutt** [kubernetes-sigs/gateway-api#1801] [Announcements]
We [the maintainers] have been building a project board which is intended to capture the important features and areas of focus which need to be resolved for our first GA release (e.g. `v1` of `GatewayClass`, `Gateway` and `HTTPRoute` and `v1.0.0` of our Golang library):

[Project Board: The Road To GA](https://github.com/orgs/kubernetes-sigs/projects/30/views/1)

Issues on this board include everything from the `v0.6.x` milestones to `v0.7.0` and then finally `v1.0.0` and are supposed to provide a sort-of "at a glance" view of what is left to do for GA. We don't yet have a specific target date for GA, but that is something we're working towards and this board helps with.

As it relates to this board you may see some issues that are out of place and you may see us moving things and bouncing milestones around on some issues a bit in upcoming weeks: that's because we still haven't finished our initial grooming and we'll be doing a bit more adjusting over the coming weeks. However we figured we would make it available for everyone to see as the bird's eye view might be helpful to some people, and there also might be times it would be helpful to review at the community meeting.

> **NOTE**: Right now we have a `Triage` column on the board which we're _temporarily_ using as a place to triage issues,
> but this will go away once we resolve https://github.com/kubernetes-sigs/gateway-api/issues/1800.

**Comment by wimnat** [comment]


**Reply by robscott** [reply]
Gateway API released v1.0 in October: https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.0.0. Envoy Gateway follows separate versioning and release schedules though, so you may want to check in with that project if you're waiting for their v1.0 release.

**Reply by wimnat** [reply]
Ah I was writing on the wrong browser tab!  Envoy Gateway also have a board titled "[Road to GA](https://github.com/orgs/envoyproxy/projects/7)" and I got confused.  Sorry about that.
